source/common/runtime/runtime_impl.cc

```diff
@@ -172,10 +174,34 @@ const std::string& SnapshotImpl::get(const std::string& key) const {
   if (entry == values_.end()) {
     return EMPTY_STRING;
   } else {
-    return entry->second.string_value_;
+    return entry->second.raw_string_value_;
   }
 }
 
+bool SnapshotImpl::featureEnabled(const std::string& key,
+                                  envoy::type::FractionalPercent default_value) const {
+  return featureEnabled(key, default_value, generator_.random());
+}
+
+bool SnapshotImpl::featureEnabled(const std::string& key,
+                                  envoy::type::FractionalPercent default_value,
```

Comment:
Nit: should this be passed `const&` here and in the interfaces above?

Reply:
Ah, good catch. Fixing.